### ISSUE TITLE
HEIC image metadata is usually ignored

### ISSUE THREAD

**Issue by cyp0633** [photoview/photoview#787]
**Describe the bug**
I imported an HEIC image into Photoview, but most of the time, the metadata is not extracted.

**To Reproduce**
1. Place any amount of HEIC image with metadata into Photoview image folder
2. Head to Photoview WebUI -> Settings -> scan the corresponding folders of the user, and wait until finish
3. Find the image identified above, see detail
4. Metadata is not displayed, and the date is the date of file creation (but not shot) (see screenshot 1)
5. Query Photoview database, only to find `exif_id` is null for that `media` entry in most scenario

**Expected behavior**
Display the metadata of all HEIC images with metadata correctly.

**Screenshots**
![no-metadata](https://user-images.githubusercontent.com/16500188/212312803-9be1c179-ff74-45ce-9cb7-31c32602cd50.png)
Fig 1 - Photoview on the right doesn't display any metadata, while Windows Explorer on the left can get it right.

![heic-without-exif-id](https://user-images.githubusercontent.com/16500188/212314069-7ff6f08d-72d2-4cf7-afac-50fce8a4554b.png)
Fig 2 - The entry of the photo in Fig 1. `exif-id` is null.

**Your environment:**
 - Are you using Docker? [no]
 - Database [MariaDB 10.9.4]
 - Browser [Firefox 109.0]
 - Version [2.3.13 (AUR)]
 - OS [Arch Linux]

**Additional context**
The original HEIC photo with metadata I've mentioned above: 
[example.zip](https://github.com/photoview/photoview/files/10411230/example.zip)

**Comment by WindLi001** [comment]
I try the example you provide, and the metadata is dispied correctly as below
![image](https://user-images.githubusercontent.com/85207398/219298228-5cf765f3-4bf8-4258-8b5a-18c3525ce931.png)

Did you install exiftool? Without it, photoview will parse metadata from photo with an internal parser, which is weaker than exiftool.

**Comment by cyp0633** [comment]
I've installed perl-image-exiftool package from pacman. If I run it manually, it prints the file exif correctly.
![图片](https://user-images.githubusercontent.com/16500188/219938249-a5cbd347-6fcd-4779-b936-29c8811fca44.png)

**Comment by WindLi001** [comment]
Maybe you should try exiftool binary.
Photoview uses package go-exiftool  [https://github.com/barasher/go-exiftool] to parse metadata from media, which is a go wrapper of exiftool. According to go-exiftool's ReadMe,
_By default, go-exiftool binary will look for **exiftool binary** in $PATH, but another location can be specified (see SetExiftoolBinaryPath functional option)._
so, the perl-image-exiftool package, which is a Perl but no binary package of exiftool, maybe NOT enough. 
Because I don't use either Arch Linux or pacman, I cannot give you the details of how to install exiftool binary on Arch Linux with pacman.

**Comment by cyp0633** [comment]
I've finally found the solution. I manually added exiftool path to env file, and exif of HEIC images can be successfully extracted now. This seems to be an AUR package problem, but not the software itself.
Many thanks for your investigation into the issue!
PS: Actually the deb package is not a binary either; it's a Perl script, just like the Arch Linux one. My exiftool installation on Arch passed go-exiftool unit test.
